src/lib/viewers/media/__tests__/Settings-test.js

```diff
@@ -101,6 +101,47 @@ describe('lib/viewers/media/Settings', () => {
         });
     });
 
+    describe('setMenuContainerDimensions', () => {
+        it('should add padding to settingsEl based on menu contents and additional padding', () => {
+            const menuEl = document.createElement('div');
+            menuEl.appendChild(document.createElement('span'));
+            settings.setMenuContainerDimensions(menuEl);
+
+            expect(settings.settingsEl.style.width).to.equal('18px');
+        });
+
+        it('should add extra padding to settingsEl based on menu contents that require scroll bar', () => {
+            const menuEl = document.createElement('div');
+            // Greater than enough to add a scroll bar.
+            menuEl.appendChild(document.createElement('span'));
```

Comment:
Add one element and set it's size with js?

Reply:
This will work especially since we're no longer counting child heights, and using offsetHeight